acceptance/lib/puppet/acceptance/common_utils.rb

```diff
@@ -174,5 +182,32 @@ def gem_command(host, type='aio')
       end
       module_function :gem_command
     end
+
+    module ManifestUtils
+      def resource_manifest(resource, title, params = {})
+        params_str = params.map do |param, value|
+          # This is not quite correct for all parameter values,
+          # but it is good enough for most purposes.
```

Comment:
Not sure I agree with adding this shared helper for such a small task when it's not correct in all situations, but I don't feel super strongly about it

Reply:
I think it should be fine. We use these enough in our tests that it is good enough to centralize them so they're there + people can update/fix it as they write tests so that it gets more and more correct.